src/Services/ActionManager.vala

```diff
@@ -158,6 +170,13 @@ public class Akira.Services.ActionManager : Object {
         typing_accelerators.set (ACTION_DELETE, "Delete");
         typing_accelerators.set (ACTION_DELETE, "BackSpace");
         typing_accelerators.set (ACTION_TOGGLE_PIXEL_GRID, "<Shift>Tab");
+
+        typing_accelerators.set (ACTION_ALIGN_LEFT, "<Control>3");
+        typing_accelerators.set (ACTION_ALIGN_HCENTER, "<Control>4");
+        typing_accelerators.set (ACTION_ALIGN_RIGHT, "<Control>5");
+        typing_accelerators.set (ACTION_ALIGN_TOP, "<Control>6");
+        typing_accelerators.set (ACTION_ALIGN_VCENTER, "<Control>7");
+        typing_accelerators.set (ACTION_ALIGN_BOTTOM, "<Control>8");
```

Comment:
Little nit: I'd prefer these to use the `Alt` modifier, and start from 1 to 6.
The `Ctrl` + number modifier should be reserved for quick zooms (eg. zoom to fit, zoom to item, etc.)